contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3687,6 +3688,102 @@ get_insert_bulk_kilobytes_per_batch()
 	return insert_bulk_kilobytes_per_batch;
 }
 
+static int
```

Comment:
Extern functions are static??

Reply:
All these `exec_*` functions are implemented the same way if you see [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/0a3b2dca475894779afb8ebe302ba21bd411aae3/contrib/babelfishpg_tsql/src/pl_exec-2.c#L40-L48). I kept it consistent.